src/Lecture2.hs

```diff
@@ -185,7 +250,10 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing [] = True
+isIncreasing (_:[]) = True
+isIncreasing (x1:x2:xs) = if x1 <= x2 then isIncreasing (x2:xs)
+                          else False
```

Comment:
You can implement `isIncreasing` in a slightly more elegant way by relying on laziness in Haskell 🦥 

```suggestion
isIncreasing (x1 : x2 : xs) = x1 < x2 && isIncreasing (x2 : xs)
```